### ISSUE TITLE
A11y_AzureNotebooks_Jupyter_MenuBar_Keyboard: Unclear functionality of drop-down present beside "Run" button.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [jupyter/notebook#3979]
**Environment Details:**
Application Name: AzureNotebooks_Jyupter
Login Url: notebooks.azure.com
Browser Details:
Edge: Microsoft Edge 41.16299.611.0
Operating System:Windows10

**ReproSteps:**
1. Login with the appropriate credentials to azure notebook account.    
2. Navigate to any of the file names present in the samples-->>click in the file name-->Welcome page opens with menubar.
3. Start Navigating through Menubar using Keyboard.
4. Navigate to Dropdown present on left side of "Open the command Palette" button.
5. Verify Unclear functionality of drop-down.

**Actual:**
**Issue1:** When navigating inside dropdown, if we try to close the dropdown without selecting any value or try to move to other Control using "Tab" key by remaining focus on any of the list item of dropdown Except the Default selected list item then that list item is automatically getting selected. 
**Issue2:** Moreover, when user select any item from it the focus directly jumps on the text field below and the content gets updated which is not announced.
**Issue3:** Keyboard functionality is different from mouse for Save, add, cut etc...Open the command palette & EnterExitRiseSlideshow controls which were there on Menubar.
For Example: When we click by Mouse on "Save" control then Notebook saved message popups on Menubar, but when we perform action by Keyboard on "Save" control then action performs but keyboard focus jumps on text field and no message popups on Menubar.

**Note:**
Same issue is observed in "**Chrome**".

Same issue is observed for "**Narrator**" and "**NVDA**" also. 

**Expected:**
**Issue1:** When navigating inside dropdown, if we try to close the dropdown without selecting any value or try to move to other Control using "Tab" key by remaining focus on any of the list item of dropdown Except the Default selected list item then that list item should not automatically get selected. 
**Issue2:** Moreover, when user select any item from it the focus directly jumps on the text field below and the content gets updated which should be announced.
Issue3: **Keyboard and Mouse Functionality should be same for all the controls present on Menu bar.**

**User Impact:**
1. While trying to close the dropdown, if value from dropdown gets automatically selected then Keyboard users face difficulty.
2. Dynamic changes are not able to understand by the screen reader users.

**Attachment:**
(https://github.com/jupyter/notebook/files/2400282/A11y_AzureNotebooks_Jupyter_MenuBar_Keyboard.pptx)

**Comment by jtpio** [comment]
Closing as there are now different issues tracking accessibility improvements for Jupyter Notebook 7, for example https://github.com/jupyter/notebook/issues/6800

Thanks!
